mmdet/models/necks/bifpn.py

```diff
@@ -0,0 +1,222 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+from mmcv.runner import BaseModule
+from ..builder import NECKS
+
+
+class SingleBiFPN(BaseModule):
+    """
+    Single layer of BiFPN
+    """
+    def __init__(self,
+                 in_channels_list,
+                 out_channels,
+                 conv_cfg=None,
+                 norm_cfg=None,
+                 act_cfg=None):
```

Comment:
Can we add arguments like FPN to conveniently control this function?
like `num_outs`, `start_level`, `end_level`, `in_channels`, `out_channels`

Reply:
Hey, thanks for the comment. Currently there still has a ton of work to do with this implementation. However, I am in the competition and the due is around the corner, so I made the PR before further polishing. Most importantly, I haven't had a decent result yet (it is still training on a single GPU). So I think I will address these comments once I have some meaningful results. Thanks again.